apps/consumers.py

```diff
@@ -38,6 +42,15 @@ def render_async_job(self, job_id):
         return JSONRenderer().render({'event': EventTypes.asyncjob_update.name,
                                       'data': AsyncJobSerializer(job).data}).decode()
 
+    async def tracking_data_save(self, event):
+        tracking_data_json = await database_sync_to_async(self.render_async_tracking_data)(event['tracking_data_id'])
+        await self.send(tracking_data_json)
+
+    def render_async_tracking_data(self, data_id):
+        data = TrackingData.objects.get(id=data_id)
+        return JSONRenderer().render({'event': EventTypes.new_tracking_data.name,
+                                      'data': data.as_point_feature}).decode()
```

Comment:
@johnkorzhuk, would you prefer tracking data updates over websockets to come back as GeoJSON Features (with points in them), or just individual Points?

Reply:
I spoke with John and he agrees that the GeoJSON Feature point might be suitable for this case.